aoservicelisting.civix.php

```diff
@@ -76,6 +76,102 @@ public static function findClass($suffix) {
     return self::CLASS_PREFIX . '_' . str_replace('\\', '_', $suffix);
   }
 
+  public static function verifyListing($duration) {
+    // Get a list of service listings who have been approved with the latest approval activity.
+    // Duration is in weeks, so multiply by 4.345
+    $weeks = (round) ($duration * 4.345) - 1;
```

Comment:
@Edzelopez why do we need to do the multiplication here? if we are passing in a numerical number of weeks when your setting the Interval to WEEK in the mysql query

Reply:
$duration contains number of months.